plugins/collaajUpdatePlugin/services/CollaajUpdateService.php

```diff
@@ -0,0 +1,96 @@
+<?php
+/**
+ *
+ * @service collaajini
+ * @package plugins.collaajini
+ * @subpackage api.services
+ */
+class CollaajUpdateService extends KalturaBaseService
+{
+    /**
+     *  Return if there is a newer update according to the givwn OS and version
+     *
+     * @action clientUpdates
+     * @param string $os
+     * @param string $version
+     * @return KalturaCaptureSpaceUpdateResponse
+     * @throws exception
+     */
+    function clientUpdatesAction ($os, $version)
+    {
+        $collaajini = new collaajini();
+        $returned_object = new KalturaCaptureSpaceUpdateResponse();
+        $result = $collaajini->returnUpdateFileUrl($os, $version);
+
+        if ($result) {
+            $res = new KalturaCaptureSpaceUpdateResponse();
+            $info = new KalturaCaptureSpaceUpdateResponseInfo();
+            $info->url = $result['url'];
+            $info->hash = new KalturaCaptureSpaceUpdateResponseInfoHash();
```

Comment:
If hash contains a KalturaCaptureSpaceUpdateResponseInfoHash then change it's type definition, currently it's defined as string.